native-image/include-metadata/run.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -ex
+
+## Run Maven Project
+# Default Configuration
+./mvnw -DskipTests clean package
+./mvnw -DskipTests -Pnative package
+./target/h2example
+
+# With Tracing Agent
+./mvnw clean
+./mvnw -Pnative -DskipTests -DskipNativeBuild=true package exec:exec@java-agent
+./mvnw -Pnative -DskipTests package exec:exec@native
```

Comment:
This NI build finishes with a failure https://github.com/graalvm/graalvm-demos/actions/runs/12258439123/job/34198207655?pr=315 
The image is created and runs we expected, but the build is “failed” when tested locally.